src/element/Hyperlink.tsx

```diff
@@ -118,7 +122,7 @@ export const Hyperlink = ({
   }, [setAppState, element, isEditing]);
 
   const onEdit = () => {
-    trackEvent("link", "Update");
+    trackEvent("hyperlink", "edit");
```

Comment:
let's distinguish between an `update` and `edit`. In here we're tracking when a user clicks on the edit button, but doesn't necessarily update. 

I didn't add a separate `update` event as I don't think it's useful knowledge. Technically even the edit isn't, if we don't distinguish how the user edits (i.e. which button they used), so we wouldn't be able to figure out which UI is being used more often.

We should prob remove this altogether or track the actual UI button.